packages/dashboard/src/hooks/useTokenInfo.ts

```diff
@@ -17,14 +17,11 @@ export const useTokenInfos = () =>
     queryFn: async () => {
       async function fetchJupiterTokens() {
         const tokenInfos: TokenInfo[] = await (
-          await fetch("https://token.jup.ag/strict")
```

Comment:
want to include lst?

Reply:
yeah need, i thought verified will have lsts too, updated